processing/src/main/java/io/druid/query/topn/PooledTopNAlgorithm.java

```diff
@@ -184,6 +199,7 @@ protected void scanAndAggregate(
 
     final int aggSize = theAggregators.length;
     final int aggExtra = aggSize % AGG_UNROLL_COUNT;
+    final IntBox currentPosition = new IntBox(0);
```

Comment:
@binlijin if it's not any faster than atomicinteger, then let's stick with atomicinteger